<launch>

	<node pkg="huanyu_joy" type="huanyu_joy_node" name="joy_controller" output="screen">
  	  <param name="joystick_device"     type="string" value="/dev/input/js0"/>  
  	  <param name="maxLinear_x"    type="double" value="0.3"/>
  	  <param name="maxLinear_y"    type="double" value="0.3"/>
      <param name="maxAngular_z"   type="double" value="1.5"/> 
	</node>

</launch>


